### PR TITLE
docs: issue #14076 updating CLI documentation

### PR DESCRIPTION
**Pull request by wolffles**
https://github.com/electron/electron/issues/14076

notes: based off of the information commented on the issue, I went ahead and started the pull request and we can build from there. All I added was some extra information in the CLI that could be useful for new electron users.

### CODE REVIEW

File: default_app/main.js [hunk 1]
```diff
@@ -177,20 +177,33 @@ if (option.file && !option.webdriver) {
...
-  - http://, https://, or file:// URL.
+
+When using a path that contains a package.json file, Electron will set:
+  - Set the appVersion
+  - Set the appName
+  - Set the userData directory path
+  - Set the userCache directory path
+  - Set the app path
+
+*IMPORTANT* FILE MODE: If you pass just a path that is in a directory without a package.json the app starts but doesn't set the above settings. This means many apis won't work properly.
```
Comment (MarshallOfSound) [inline default_app/main.js]: > This means many apis won't work properly.

This isn't entirely accurate, all APIs do work properly.

Comment (wolffles) [reply default_app/main.js]: I see, misinformed, I can remove this part no prob, any more ideas to add?

**Comment by wolffles** [comment]
I'm confused why these tests are failing, can someone help me understand.

**Comment by miniak** [comment]
@wolffles please rebase

File: default_app/main.js [hunk 2]
```diff
@@ -166,20 +166,29 @@ if (option.file && !option.webdriver) {
...
+  - Set the app path
 
 Options:
   -i, --interactive     Open a REPL to the main process.
   -r, --require         Module to preload (option can be repeated).
   -v, --version         Print the version.
-  -a, --abi             Print the Node ABI version.`
+  -a, --abi             Print the Node ABI version.
+  
+For more information about useage please visit: https://electronjs.org/docs`
```
Comment (miniak) [inline default_app/main.js]: typo useage -> usage

**Pull request closed without merging**